### ISSUE TITLE
IPv6 Address as Server URL

### ISSUE THREAD

**Issue by jan-kl** [nextcloud/ios#2684]
### Steps to reproduce

1. Start process to add a new account
2. Enter IPv6 as the server URL(i.e. https://[2001:db8:3333:4444:5555:6666:7777:8888])

### Expected behaviour

Establish a connection to the server running at the specified IPv6 Address.

### Actual behaviour

An error message is shown stating that the URL is invalid.

### Screenshots

![image](https://github.com/nextcloud/ios/assets/55736760/6681e081-04b4-4739-b081-8ddd97018230)

![image](https://github.com/nextcloud/ios/assets/55736760/b2e91573-1051-4191-a5e7-6607505b5ce7)

### Reasoning or why should it be changed/implemented?

There are a two main reasons why it would be nice to be able to enter an IPv6 address. First of all, not everyone has a public IPv4 address. Also IPv6 addresses are not scanned for open ports as regular as IPv4 addresses are. Registering the IP at a DNS provider would also make the IPv6 more exposed.

### Environment data

**iOS version:** 17.1.1

**Nextcloud iOS app version:** 4.9.2.20

**Comment by jan-kl** [comment]
Unfortunately I was not able to solve the problem myself, but I hope the issue is easily solvable.

**Comment by yywe** [comment]
saw the same issue. @jan-kl did you get it resolved ???

**Comment by jan-kl** [comment]
No, as you can see here I never got any reaction and for now accepted, that it is just not possible to use IPv6 addresses.
